pkg/builder/controller.go

```diff
@@ -58,6 +59,8 @@ type Builder struct {
 	ownsInput        []OwnsInput
 	watchesInput     []WatchesInput
 	mgr              manager.Manager
+	cluster          cluster.Cluster
+	clusterName      string
```

Comment:
I know the PR is WIP and I'm just taking a look to get an overview.

The ClusterName field is never written atm.

Reply:
Updated the PR. This is not relevant anymore.